### PR TITLE
chore: fix audit findings

### PR DESCRIPTION
**Pull request by bitcoinbrisbane**
[Link to ticket](url)

- One liner ticket title/description
- If not intuitive from the ticket -> short summary of code changes
- Include screenshots for visual UI changes

### CODE REVIEW

**Review by brettvniekerk — DISMISSED** [review]

File: tests/market.test.ts [hunk 1]
```diff
@@ -41,6 +41,7 @@ describe("Market", () => {
 	const MARGIN = 100;
 	const TIMEOUT_DAYS = 5;
 	const WINNER = 0x01;
+	const LOOSER = 0x02;
```
Comment (bazmatic) [inline tests/market.test.ts:44]:
```suggestion
	const LOSER = 0x02;
```

File: tests/market.test.ts [hunk 2]
```diff
@@ -1081,6 +1082,88 @@ describe("Market", () => {
...
+				.withArgs(
+					index,
+					formatBytes16String(propositionId),
+					formatBytes16String(marketId),
+					wager,
+					272727300,
+					bob.address
+				);
+
+			// add a looser result
```
Comment (bazmatic) [inline tests/market.test.ts:1139]:
```suggestion
			// add a loser result
```

File: tests/market.test.ts [hunk 3]
```diff
@@ -1081,6 +1082,88 @@ describe("Market", () => {
 			expect(await market.getTotalExposure()).to.equal(0);
 		});
 
+		it("Should not payout wage after timeout / expire has been reached and result added to the oracle", async () => {
```
Comment (bazmatic) [inline tests/market.test.ts:1085]:
```suggestion
		it("Should not payout wager after timeout / expire has been reached and result added to the oracle", async () => {
```

File: tests/market.test.ts [hunk 4]
```diff
@@ -1081,6 +1082,88 @@ describe("Market", () => {
...
+			);
+
+			await hre.network.provider.request({
+				method: "evm_setNextBlockTimestamp",
+				params: [end + 31 * 24 * 60 * 60]
+			});
+
+			expect(await market.settle(index), "Should emit a Settled event")
+				.to.emit(market, "Settled")
+				.withArgs(index, 272727300, LOOSER, vault.address);
```
Comment (bazmatic) [inline tests/market.test.ts:1160]:
```suggestion
				.withArgs(index, 272727300, LOSER, vault.address);
```

File: tests/market.test.ts [hunk 5]
```diff
@@ -1081,6 +1082,88 @@ describe("Market", () => {
...
+
+			expect(await market.getInPlayCount()).to.equal(0);
+			expect(await market.getTotalInPlay()).to.equal(0);
+			expect(await market.getTotalExposure()).to.equal(0);
```
Comment (bazmatic) [inline tests/market.test.ts:1164]: Suggest checking the balance of the punter to show they didn't get paid, and/or balance of vault to show that they did.

**Review by deBhal — CHANGES_REQUESTED** [review]
Looks like the underlying code has changed to:  
_market.test.ts:1016_

> it("Should payout wage after timeout / expire has been reached", async () => {

but that the patch is still appropriate

**Pull request merged**